src/config/lang-config.interface.ts

```diff
@@ -0,0 +1,7 @@
+import { Config } from './config.interface';
+
+export interface LangConfig extends Config {
```

Comment:
Please add a comment/TypeDocs to describe this new interface.

Reply:
done